git/git-plan.md

```diff
@@ -0,0 +1,88 @@
+# Version Control with Git & GitHub
+
+## Introduction
+
+- what is version control?
+- why should I use it?
+- what is git?
+- what is GitHub?
+
+## GitHub
+
+- look at an example repo
+    - e.g. this one? One of the Software Carpentry lessons? any other suggestions?
```

Comment:
yeah why not this one. Has a README, has LICENSE, has decent amount of commits.

Reply:
👍